scripts/db_migrations/v2.20/upload_models_s3.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import uuid
+from pymongo import MongoClient
+import boto3
+import re
+
+##### Connect to AWS S3 #####
+s3 = boto3.client(
+  's3',
+  aws_access_key_id='AKIAJIJNXOYFX5B6SHXA',
```

Comment:
I don't mean putting the actual keys into the script, i mean putting it in as a parater to the script (like the mongo connections) lol